packages/frontend/app/utils/config.server.ts

```diff
@@ -0,0 +1,3 @@
+export const timeoutTwoPhase = BigInt(
+  process.env.WITHDRAWAL_TIMEOUT_SECONDS || 0 /*2-phase disabled*/
+)
```

Comment:
similar to my comment earlier, I think all admin UI mutations for liquidity should just be single-phase, and we can hard code 0 in those calls.

Reply:
Ah, ok, sorry. I misunderstood @mkurapov . Fixed now.